R/lockbox.R

```diff
@@ -132,8 +132,12 @@ lockbox_download_dir <- function() {
 
 #' The transient lockbox library path.
 lockbox_transient_dir <- function() {
-  getOption("lockbox.transient_dir") %||%
+  if (!is.null(.lockbox_env$session_id)) {
```

Comment:
@drknexus Good point. I didn't like the complexity of following `%||%` with `ifelse` but I'll re-order this a bit.